mcux/mcux-sdk/drivers/mipi_dsi_split/fsl_mipi_dsi.c

```diff
@@ -538,7 +538,7 @@ void DSI_SetDpiConfig(const MIPI_DSI_Type *base,
     assert(NULL != config);
 
     /* coefficient DPI event size to number of DSI bytes. */
-    float coff = ((float)numLanes * (float)dsiHsBitClkFreq_Hz) / ((float)dpiPixelClkFreq_Hz * 8.0);
+    float coff = ((float)numLanes * (float)dsiHsBitClkFreq_Hz) / ((float)dpiPixelClkFreq_Hz * 8.0f);
```

Comment:
Have we submitted this change to the SDK team?

Reply:
I will submit all my patches as PRs to the SDK once we are clean and merged